### DISCUSSION TITLE
Copilot activation failed in VScode

### DISCUSSION THREAD

**Discussion by 0412jason** [community/community#14109] [Copilot Conversations] [answered]
I follow the guide on Github, and I have join the waitlist.
But I still get some error.
How can I fix it?

This is my Output log:
```
[INFO] [auth] [2022-04-02T16:18:04.124Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-04-02T16:18:04.126Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"
```

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
